test/service/job/test_job_type_controller.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+import os
+
+from run_service import app
+
+config_dir = os.environ['CONFIG_DIR']
+job_types_dir = os.path.join(config_dir, 'job_types')
+
+
+class JobTypeTest(unittest.TestCase):
+
+    def setUp(self):
+        app.testing = True
+        self.client = app.test_client()
+
+    def test_job_types_list_length(self):
+        """
+        Checks if the length of the element list of what the API returns is
+        the same than the number of files in the directory on disk.
+
+        :return: None
```

Comment:
Einigen wir uns darauf in docstrings jedes Mal return: None zu machen?

Reply:
Ich finde eigentlich, dass das weggelassen werden kann, vor allem bei Tests. Evtl. macht es Sinn, wenn man bei einer Methode, die so aussieht, als würde sie etwas zurückgeben, explizit machen will, dass das nicht der Fall ist. Aber andererseits würde ich dann wieder dafür argumentieren, dass man es am besten schon dem Methodennamen ansehen sollte, wenn das der Fall ist (z.B. `assert_...`).